src/driftpy/drift_user.py

```diff
@@ -165,46 +172,62 @@ async def set_cache(self, CACHE=None):
         user = await get_user_account(self.program, self.authority, self.subaccount_id)
         self.CACHE["user"] = user
 
+    async def subscribe(self):
+        if self.is_subscribed:
+            while self.is_subscribed:
+                await self.poll_data()
+                await asyncio.sleep()
+
+    async def unsubscribe(self):
+        self.is_subscribed = False
+
+    async def poll_data(self):
```

Comment:
1) think ideally this is turned into an interface, similar to the the typescript one, so that users can plug their own customization
2) we should probably change the drift client to handle fetching market accounts, while user just has subscriber for fetching user accounts, similar to typescript

this could also be in a follow up pr